### ISSUE TITLE
Virtual monitor not being disabled

### ISSUE THREAD

**Issue by Permanently** [fehbari/sunshine-scripts#6]
On the script, these two commands are supposed to disable the virtual display:

https://github.com/fehbari/sunshine-scripts/blob/fca52f002cf6ae295e084789ae3183b8855f4b06/stop_streaming.bat#L26-L28

However, they are instead spitting out PnPUtil help messages (Windows 10 22H2 (19045.5011)), thus not disabling this at all. I'm working on a solution to this, with PowerShell to get around this. I listed it on `pnputil /enum-devices` and found these:

```
Instance ID:                ROOT\DISPLAY\0001
Device Description:         Virtual Display Driver by MTT
Class Name:                 Display
Class GUID:                 {4d36e968-e325-11ce-bfc1-08002be10318}
Manufacturer Name:          MikeTheTech
Status:                     Started
Driver Name:                oem76.inf

Instance ID:                DISPLAY\LNX0000\1&15ecd195&1&UID256
Device Description:         Generic PnP Monitor
Class Name:                 Monitor
Class GUID:                 {4d36e96e-e325-11ce-bfc1-08002be10318}
Manufacturer Name:          (Standard monitor types)
Status:                     Started
Driver Name:                monitor.inf
```

~~Will reference PR to this when I figure it out~~ see #7

**Comment by Permanently** [comment]
i pressed enter to submit instead of enter to put a new line - editing now to add all descriptions lmao

**Comment by fehbari** [comment]
@Permanently I'm really puzzled by this, since the device ID used for enabling and disabling the virtual monitor are the same. It works fine for me in W11 23H2 (both enabling and disabling), with VDD 23.12.2.1 HDR.

I'm setting up a W10 22H2 VM to try to reproduce the issue and better understand the root cause, so bear with me please.

I saw your PR and it looks like a good approach (only a little brittle given the friendly device name might change). But if I can pinpoint the exact issue you're seeing in W10 we might be able to continue using PNPUTIL on the well-known device IDs, and avoid the more expensive subshell and Pnp device lookup. It's entirely possible the device IDs are indeed just difference between W10 and W11, but I want to make sure I fully understand this before commiting to a fix. 

Thanks for reporting this. Setups out there are varied and I'm limited by what I can test on my system, so this helps.

**Comment by fehbari** [comment]
I can confirm this is a regression in the 1.0.2 release of the scripts. `PNPUTIL` isn't working correctly. Switching back to `devcon` and requiring admin privileges again works fine in Windows 10 and 11.

**Comment by fehbari** [comment]
This should now be fixed in [v1.1.0](https://github.com/fehbari/sunshine-scripts/releases/tag/1.1.0). I've included your lookup suggestion as part of a refactor to move these scripts over to PowerShell and improve reliability. Working with Batchfile exclusively was becoming a hindrance.

Tested on Windows 11 23H2 with HDR driver, and Windows 10 22H2 with regular driver (inside a VM).

**Windows 11**

```
> cmd /c "c:\Tools\sunshine-scripts\start_streaming.bat"
Enabling virtual display...
Enabled device: IddSampleDriver Device HDR
Setting resolution with QRes: C:\Tools\QRes\QRes.exe /X:1920 /Y:1080 /R:60
Turning HDR off with HDRCmd: C:\Tools\HDRTray\HDRCmd off
Turning off G-Sync: C:\Tools\gsync-toggle\gsynctoggle 0
Setting FPS limit with frl-toggle: C:\Tools\frl-toggle\frltoggle.exe 60

> cmd /c "c:\Tools\sunshine-scripts\stop_streaming.bat"
Disabling virtual display...
Disabled device: IddSampleDriver Device HDR
Setting resolution with QRes: C:\Tools\QRes\QRes.exe /X:1920 /Y:1080 /R:60
Turning HDR off with HDRCmd: C:\Tools\HDRTray\HDRCmd off
Turning on G-Sync: C:\Tools\gsync-toggle\gsynctoggle 1
Setting FPS limit with frl-toggle: C:\Tools\frl-toggle\frltoggle.exe 57
```

**Windows 10**

![Captura de tela 2024-11-05 050224](https://github.com/user-attachments/assets/84a776a2-451d-42c2-8862-39bebf97a15d)

![Captura de tela 2024-11-05 053619](https://github.com/user-attachments/assets/13a471a1-7436-42d3-b6e8-d23246a7b5cd)

(ignore the NVAPI errors, they were caused by this running inside a VM)

**Comment by Permanently** [comment]
Brill! I’ll give this a whirl later in the day and report back on performance my side (Windows 10 22H2)
